src/app/main/time-groups/edit-time-group/edit-time-group.component.ts

```diff
@@ -212,28 +229,110 @@ export class EditTimeGroupComponent implements OnInit {
     this.subscriptions.push(
       this.timeGroupService.updateTimeGroupEmployees(this.timeGroupId, updateParams).subscribe(response => {
         this.appNotificationService.push(this.translate.instant('tr_time_group_employees_updated_successfully'), 'success');
+        this.employeesList = []
+        this.getEmployees()
```

Comment:
Why you are not adding type = 'search' instead of `this.employeesList = []`

Reply:
done, I added it